frontend/src/components/QueueTable.js

```diff
@@ -26,51 +26,60 @@ const useStyles = makeStyles(theme => ({
 const QueueTable = observer(queueData => {
   const queueStore = useContext(QueueStoreContext)
   const classes = useStyles()
+  const [visibleDialog, setVisibleDialog] = useState(false)
+  const [notesVisit, setNotesVisit] = useState(1)
+
+  const handleNotesClick = visitId => {
+    setNotesVisit(visitId)
+    toggleVisibleDialog()
+  }
 
-  const [visibleDialog, setVisibleDialog] = React.useState(false)
   const toggleVisibleDialog = () => {
     setVisibleDialog(!visibleDialog)
   }
 
+  const seenHandler = id => {
+    queueStore.updateStatus(queueData.queueData, id.id, "SEEN")
+  }
+
   const statusOptions = [
-    { value: "ARRIVED", name: "Arrived" },
-    { value: "SEEN", name: "Seen" },
+    { value: "ARRIVED", name: "Waiting" },
     { value: "STEPPED_OUT", name: "Stepped Out" },
-    { value: "CAME_BACK", name: "Came Back" },
     { value: "LEFT", name: "Left" },
   ]
+
   const urgencyOptions = [
-    { value: 1, name: 1 },
-    { value: 2, name: 2 },
-    { value: 3, name: 3 },
-    { value: 4, name: 4 },
-    { value: 5, name: 5 },
+    { value: "ONE", name: 1 },
```

Comment:
Aren't the original urgency options correct since Urgency is being turned to a int? Its also necessary for urgency ordering.

Reply:
Are you saying these should be changed to 1, 2... here and in the back end core/models/visit.py?

Reply:
Yes. From what I understood, its not suppose to be words. Also, every time urgency is sorted, viewed, or saved, it would need to be converted to and from numbers unnecessarily.

Reply:
Got it.